src/utils/RealtimeFeedUtils.js

```diff
@@ -4,11 +4,14 @@ import LogUtils from './LogUtils';
 import RequestUtils from './RequestUtils';
 import TokenUtils from './TokenUtils';
 
+const THREE_SEC_IN_MS = 3000;
+
 async function fetchRTF(): Object {
   const options = {
     method: 'GET',
     url: `http://${PYTHON_APP || 'localhost'}:5000/rtf`,
     headers: { 'Cache-Control': 'no-cache' },
+    timeout: THREE_SEC_IN_MS,
```

Comment:
can we put this in a `constants.js` similar to how pollo does it, since it's being referenced from a bunch of places, maybe like:

```
const GET_OPTIONS = { 
  method: 'GET',
  headers: { 'Cache-Control': 'no-cache' },
  timeout: THREE_SEC_IN_MS,
}
```

and then you can have:

```
const options = {
  ...GET_OPTIONS,
  url: `...`
}
```